configs/providers.js

```diff
@@ -1,7 +1,7 @@
 module.exports = [
 	{
 		name: "RIBBON",
-		domain: "https://bins.ribbon.co",
+		domain: "http://bins.ribbon.co",
```

Comment:
I will updated the Ribbon provider with the new API details which supports https